src/components/common/AppFooter/AppFooterDesktop/AppFooterDesktop.jsx

```diff
@@ -14,9 +14,12 @@ const AppFooterDesktop = () =>
     <div className="container">
```

Comment:
Use container component
```suggestion
    <Container className="app-footer-desktop__container">
```